### DISCUSSION TITLE
Are Supabase Storage Images Available With Local Development?

### DISCUSSION THREAD

**Discussion by drewbietron** [supabase/supabase#9657] [Questions] [answered]
I am unable to view images from Supabase Storage while using local development.

I am able to upload images to Supabase Storage locally, but when I try to access them from the url thats returned from [getPublicUrl](https://supabase.com/docs/reference/javascript/storage-from-getpublicurl), I get a 404.  

```
{"statusCode":"404","error":"Not found","message":"The resource was not found"}
```

I can also confirm that the images are in fact uploaded to the database, which I have to do by going to the Database tab in studio and changing the schema to storage since the Storage tab isn't there in local development Supabase.  

The buckets are good as far as RLS goes, everything works fine with the same set up in a hosted environment, its just with local development.

I'm hoping to be able to view local images and that I am missing something obvious 🤷.

Thanks

**Comment by GaryAustin1** [comment, marked as answer]
This was solved in Discord and was the public flag in the bucket table.
